sympy/polys/specialpolys.py

```diff
@@ -90,13 +90,15 @@ def cyclotomic_poly(n, x=None, polys=False):
 
 
 @public
-def symmetric_poly(n, *gens, **args):
-    """Generates symmetric polynomial of order `n`.
+def symmetric_poly(n, *gens, polys=False):
+    """
+    Generates symmetric polynomial of order `n`.
 
+    Parameters:
+    polys: bool, optional (default: False)
     Returns a Poly object when ``polys=True``, otherwise
     (default) returns an expression.
```

Comment:
```suggestion
    polys: bool, optional (default: False)
        Returns a Poly object when ``polys=True``, otherwise
        (default) returns an expression.
```